modules/lifestreetBidAdapter.md

```diff
@@ -45,3 +57,43 @@ Module that connects to Lifestreet's demand sources
         }
     ];
 ```
+
+# Test Parameters (Banner)
+```
+    const adUnits = [
+        {
+            code: 'test-ad',
+            sizes: [[160, 600]],
```

Comment:
Can you update your examples in this file to use adUnit -> mediaTypes -> banner -> sizes, like described here: http://prebid.org/blog/pbjs-3 ?